compiler-plugin/testData/api/overridee.kt

```diff
@@ -85,6 +85,8 @@
 // JavaAccessorImpl.getX() -> KtInterfaceWithProperty.x
 // JavaAccessorImpl.getY() -> KtInterfaceWithProperty.y
 // JavaAccessorImpl.setY(value:Int) -> KtInterfaceWithProperty.y
+// JavaAnno:
+// JavaAnno.intParam() -> null
```

Comment:
Just to double-check: `-> null` means that it doesn't override anything?

Reply:
yes